packages/sku/scripts/postinstall.js

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env node
 // @ts-check
 import { readFile } from 'node:fs/promises';
-import { setCwd, getPathFromCwd, cwd } from '../lib/cwd.js';
+import { setCwd, getPathFromCwd, cwd } from '../src/lib/cwd.js';
```

Comment:
If `postinstall` points to dist it must be created before it can use them. When you install the app for the first time it will run `postinstall` when you do `pnpm install` at which point the `dist` folder won't be there yet. 

Should probably do something else here.

Reply:
This is the postinstall for the `sku` package though, so it should be included alongside the package, and executed when `sku` is installed. I figured this would work fine since `dist` should exist in the package right?

Reply:
At that point, yes, it does exist. However, when you first clone + install SKU it will error out.

Reply:
Oh right. Maybe we run `build` in the repo postinstall script? That could be kind of annoying though.

Reply:
And it might only happen _after_ the sku postinstall runs, which won't work. 🤔

Reply:
we can't build before we install. we cant postinstall until we build. The dilemma...

Reply:
I believe that the postinstall script shouldn't have to run on the `sku` package install itself. If that's the case I can probably create an early return before we dynamic import everything required to run the postinstall when someone actually installs the 'sku' package as a dependency